ostree/ostree_dest.go

```diff
@@ -335,6 +335,9 @@ func (d *ostreeImageDestination) importConfig(repo *otbuiltin.Repo, blob *blobTo
 // reflected in the manifest that will be written.
 // If the transport can not reuse the requested blob, TryReusingBlob returns (false, {}, nil); it returns a non-nil error only on an unexpected failure.
 func (d *ostreeImageDestination) TryReusingBlobWithOptions(ctx context.Context, info types.BlobInfo, options private.TryReusingBlobOptions) (bool, private.ReusedBlob, error) {
+	if !impl.BlobMatchesRequiredCompression(options) {
```

Comment:
I think this also needs updating (`ostree` is not compiled by default … so we keep breaking it.)

Reply:
I did not get this comment, do you mean we should not add this to `ostree` ?

Reply:
Doesn’t this need `s/Blob/OriginalBlob/`?

Try `go test -tags containers_image_ostree ./ostree` …

… oh my. The tests have been broken … \*checks\* … consecutively since 2022-03-16, three different ways, all by me.

Reply:
> The tests have been broken

For the record, #2037 should fix that (but it doesn’t add the subpackage to CI, out of both laziness and curiosity of how long it will be broken next time).